ENTITY/IsEntityOnScreen.md

```diff
@@ -8,13 +8,13 @@ ns: ENTITY
 BOOL IS_ENTITY_ON_SCREEN(Entity entity);
 ```
 
-```
-Returns true if the entity is in between the minimum and maximum values for the 2d screen coords.   
-This means that it will return true even if the entity is behind a wall for example, as long as you're looking at their location.   
-Chipping  
-```
+Determines whether the screen position of the specified entity is within the 2D bounds of the screen.
 
-## Parameters
-* **entity**: 
+This native will not check if the entity is not visible due to being occluded (for example, behind a wall). To check if a entity is on screen and is not occluded, use [IS_ENTITY_OCCLUDED](#_0xE31C2C72B8692B64).
 
 ## Return value
+
+Returns `true` if the the entity is in between the minimum and maximum values for the 2d screen coord; otherwise, `false`.
```

Comment:
Return value need to be after param. Please refer to https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md